packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -443,7 +443,10 @@ export namespace AbstractVSXExtensionComponent {
 
 const downloadFormatter = new Intl.NumberFormat();
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const downloadCompactFormatter = new Intl.NumberFormat(undefined, { notation: 'compact', compactDisplay: 'short' } as any);
+const downloadCompactFormatter = new Intl.NumberFormat('en-US', { notation: 'compact', compactDisplay: 'short' } as any);
+const averageRatingFormatter = (averageRating: number): number => Math.round(averageRating * 2) / 2;
+const getAverageRatingTitle = (averageRating: number): string =>
+    nls.localize('theia/vsx-extension/averageRating', 'Average rating: {0} out of 5', averageRatingFormatter(averageRating));
```

Comment:
Minor: This should likely be
```suggestion
    nls.localize('theia/vsx-registry/averageRating', 'Average rating: {0} out of 5', averageRatingFormatter(averageRating));
```